client/src/components/Sidebar/modules/Search.scss

```diff
@@ -30,7 +30,7 @@
     -webkit-font-smoothing: auto;
 
     .sidebar--slim & {
-      opacity: 0;
+      @apply hover:w-cursor-pointer w-opacity-0;
```

Comment:
Could we instead set `w-cursor-pointer` without `hover:`? Just so this is a bit shorter.